File: src/main/java/io/myosotisdev/utopianism/entity/ILivingEntity.java
package io.myosotisdev.utopianism.entity;

import io.myosotisdev.utopianism.battle.attributes.Health;
import net.minestom.server.entity.LivingEntity;

public interface ILivingEntity extends IBattleEntity
{
    LivingEntity getTarget();

    Health getHealthBar();
}

File: src/main/java/io/myosotisdev/utopianism/entity/ILivingEntity.kt
package io.myosotisdev.utopianism.entity

import io.myosotisdev.utopianism.battle.attributes.Health
import net.minestom.server.entity.LivingEntity

interface ILivingEntity : IBattleEntity
{
    val target: LivingEntity
    val healthBar: Health?
}